syne_tune/optimizer/schedulers/searchers/botorch/__init__.py

```diff
@@ -16,9 +16,11 @@
 
 try:
     from syne_tune.optimizer.schedulers.searchers.botorch.botorch_searcher import (  # noqa: F401
-        BotorchSearcher,
+        BoTorchSearcher,
+        BotorchSearcher,  # deprecated
```

Comment:
Can `BotorchSearcher` be removed entirely in favor of `BoTorchSearcher`?

Reply:
I see -- looks like you are keeping it for backwards compatibility

Reply:
Yes, we can remove it after a while